src/app/components/Feedback/Feedback.jsx

```diff
@@ -1,62 +1,107 @@
+/* global alert */
 import React from 'react';
 import PropTypes from 'prop-types';
 import FocusTrap from 'focus-trap-react';
+import axios from 'axios';
 
 import { trackDiscovery } from '../../utils/utils';
+import appConfig from '../../data/appConfig';
+
+const initialFields = {
+  Email: '',
+  Feedback: '',
+};
 
 class Feedback extends React.Component {
   constructor(props) {
     super(props);
 
-    this.state = { showForm: false };
+    this.currentURL = (this.props.location.pathname +
```

Comment:
This seems fine to add to the constructor, but I don't think it makes sense to add it to the state. It's never actually being updated, even with `setState`. In the `setState` call, you're still passing in `this.currentURL` so it's technically not changing. Keeping it as an instance variable is enough.

Reply:
Granted, I haven't tested this but if it's mounting for every page route, the URL should update accordingly.